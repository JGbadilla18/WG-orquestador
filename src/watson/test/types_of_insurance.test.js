const testWatson = require('./testWatson');

watsonText = 'Tipos de seguros.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
