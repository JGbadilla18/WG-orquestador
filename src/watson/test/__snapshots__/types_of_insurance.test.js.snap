// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tipos de seguros. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Conoce más aquí",
          "url": " http://seguros.elroble.com/productos/",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Puedes comunicarte a través de Whatsapp con Seguros El Roble, agrega este número y haz tu pregunta: 2420-3397",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
